File: 2022/programmazione/cover/cover.py
#!/bin/env python3

import sys

# Se vuoi leggere/scrivere da file decommenta qua
# fin = open("input.txt", "r")  # File di input fornito dalla piattaforma
# fout = open("output.txt", "w")  # File di output fornito dalla piattaforma

# Se vuoi leggere/scrivere da linea di comando decommenta qua
fin = sys.stdin  # File di input fornito dalla piattaforma
fout = sys.stdout  # File di output fornito dalla piattaforma


def conta(N, K, ranges):
    # SCRIVI QUA IL TUO CODICE
    return 0


N, K = map(int, fin.readline().strip().split(" "))
ranges = []

for _ in range(N):
    start, end = map(int, fin.readline().strip().split(" "))
    ranges.append([start, end])

print(conta(N, K, ranges), file=fout)

File: 2022/programmazione/cover/cover.cpp
#include <stdio.h>
#include <iostream>
#include <vector>
#include <array>

using namespace std;

int conta(int N, int K, const vector<array<long long int, 2>> &ranges) {
    // SCRIVI QUA IL TUO CODICE
    return 0;
}

int main() {

    // freopen("input.txt", "r", stdin); // DECOMMENTA QUA SE VUOI LEGGERE DA FILE
    // freopen("output.txt", "w", stdout); // DECOMMENTA QUA SE VUOI SCRIVERE DA FILE

    int N, K;
    cin >> N >> K;
    vector<array<long long int, 2>> ranges(N);

    for(int i=0; i<N; i++) {
        cin >> ranges[i][0];
        cin >> ranges[i][1];
    }

    cout << conta(N, K, ranges);

    return 0;
}
